apps/search/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.17.2",
   "scripts": {
     "build": "pnpm generate && next build",
+    "deploy": "npx tsx scripts/migrations/run-webhooks-migration.ts && pnpm run build",
```

Comment:
@witoszekdev using already implemented migration from some time ago. It just updates webhooks to the newest versions, without applying patches sequentially. 

In the future We'll likely introduce migrations but here I'd like to make it quick.